contracts/DxMgnPool.sol

```diff
@@ -125,4 +145,18 @@ contract DxMgnPool {
     function hasPoolingEnded() private view returns (bool) {
         return block.number > poolingPeriodEndBlockNumber;
     }
+
+    function assignBuyAndSellTokens() 
```

Comment:
That's very weird syntax, I'm surprised it works. Can we use `return` and mark this method as `private view`